SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Anbox Cloud security policy
```

Comment:
Should we put the same into our documentation given that most components aren't open source and wont have a SECURITY.md?

Reply:
Couldn't hurt to put it in docs. I will create a PR for that.